.test/src/test/kotlin/ProjectConsistencyTests.kt

```diff
@@ -242,14 +250,53 @@ class ProjectConsistencyTests {
             return "$repoRootDirectoryFile/$projectDirectoryName/app/src/main/java/$packagePath"
         }
 
+        /**
+         * Return project name without the language
+         * Java
+         */
+        private fun getProjectName(projectDirectoryName: String) = projectDirectoryName
+                .replace("-Java", "")
+                .replace("-Kotlin", "")
+
+        /**
+         * Return main readme link name
+         * Java
+         */
```

Comment:
Does "Java" add any value?
```suggestion
        // Return main readme link name.
```